catroid/src/main/java/org/catrobat/catroid/content/bricks/StitchBrick.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content.bricks;
+
+import android.content.Context;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.actions.ScriptSequenceAction;
+
+import java.util.List;
+
+public class StitchBrick extends BrickBaseType {
+
+	private static final long serialVersionUID = 1L;
+
+	public StitchBrick() {
+	}
+
+	@Override
+	public View getPrototypeView(Context context) {
+		return super.getPrototypeView(context);
+	}
+
+	@Override
+	public Brick clone() {
```

Comment:
why don't you call ```super.clone()``` here? the ```clone()``` method should obtain a copy of the object from its super, as specified [here](https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#clone), or more with more examples [here](https://en.wikipedia.org/wiki/Clone_(Java_method)). You should be careful about stuff like this because it can break many things (e.g. the commentedOut state will be incorrect if you just create a new instance instead of properly duplicating the brick).